downstairs/src/region.rs

```diff
@@ -4418,10 +4409,83 @@ pub(crate) mod test {
         .concat();
 
         // write_unwritten = true
-        region.region_write(&writes, JobId(0), true).await?;
+        region.region_write(&writes, JobId(0), true).await.unwrap();
 
         // Validate written data by reading everything back and comparing with
         // data buffer
-        validate_whole_region(&mut region, &data).await
+        validate_whole_region(&mut region, &data).await;
+    }
+
+    macro_rules! region_test_suite {
```

Comment:
We should put some comments here to instruct future us that if we wish to add a region test, it should be added to this list here (as, I assume that is what we need to do).

Reply:
Good idea, added in d06451a93a90ecb656faea5a5cd809c473c60dad